static/js/index.tsx

```diff
@@ -1,14 +1,16 @@
-import React from "react"
+import React, { ComponentType } from "react"
 import ReactDOM from "react-dom"
 import { AppContainer } from "react-hot-loader"
 import { createBrowserHistory } from "history"
 
-import configureStore from "../store/configureStore"
-import Router, { routes } from "../Router"
+import configureStore from "./store/configureStore"
+import Router, { RootProps } from "./Router"
 
 import * as Sentry from "@sentry/browser"
-// Object.entries polyfill
-import entries from "object.entries"
+
+// polyfill for ES5 support :/
```

Comment:
Do we need this? It seems like modern browsers support ES6, though I don't know if that means every part https://caniuse.com/?search=es6

Reply:
maybe it is finally time - we talked about dropping support for IE11. I guess for this project support for old browsers is even less important, since people are going to be using it on new computers at work.

Reply:
I'll go ahead and remove the polyfills, we can add them back in the future if need be